helm_deploy/values-dev.yaml

```diff
@@ -33,8 +33,9 @@ generic-service:
     MANAGE_SOC_CASES_UI_URL: "https://manage-soc-cases-dev.hmpps.service.justice.gov.uk"
     WELCOME_PEOPLE_INTO_PRISON_UI_URL: "https://welcome-dev.prison.service.justice.gov.uk"
     CREATE_AND_VARY_A_LICENCE_UI_URL: "https://create-and-vary-a-licence-dev.hmpps.service.justice.gov.uk"
+    MANAGE_ADJUDICATIONS_API_URL: "https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk"
     CALCULATE_RELEASE_DATES_UI_URL: "https://calculate-release-dates-dev.hmpps.service.justice.gov.uk"
-    FEEDBACK_DISABLED_PRISONS: "LEI,FNI,RSI"
+    FEEDBACK_DISABLED_PRISONS: "LEI,FNI"
```

Comment:
This updated by https://github.com/ministryofjustice/hmpps-prisoner-profile/pull/241, git struggled with the double revert

Reply:
@neilmendum Can you try merging in latest `main` and see if it resolves this.